pom.xml

```diff
@@ -100,6 +100,7 @@
         <hibernate-orm.plugin.version>6.2.13.Final</hibernate-orm.plugin.version>
         <hibernate.c3p0.version>6.2.13.Final</hibernate.c3p0.version>
         <infinispan.version>15.0.5.Final</infinispan.version>
+        <protostream.version>5.0.6.Final</protostream.version>
```

Comment:
Suggestion: Create a follow-up issue to align the version of protostream here with the one used in Infinispan once it catches up, and then remove the explicit dependency management we use below.

Reply:
I've created #31032 to track this